kyuubi-server/pom.xml

```diff
@@ -385,6 +385,13 @@
             <scope>test</scope>
         </dependency>
 
+        <dependency>
+            <groupId>org.mockito</groupId>
+            <artifactId>mockito-core</artifactId>
+            <scope>test</scope>
+            <version>3.4.6</version>
```

Comment:
can we add it to dependencyManagement then inherit it here？

Reply:
ok, good idea.

Reply:
Please use scalatestplus-mockito instead.
```
<properties>
    <scalatestplus.version>3.2.9.0</scalatestplus.version>
</properties>

<dependency>
    <groupId>org.scalatestplus</groupId>
    <artifactId>mockito-3-4_${scala.binary.version}</artifactId>
    <version>${scalatestplus.version}</version>
</dependency>
```